app/src/main/java/org/metabrainz/android/presentation/features/dashboard/DashboardActivity.kt

```diff
@@ -102,7 +103,7 @@ class DashboardActivity : AppCompatActivity() {
             items.forEach { item ->
                 BottomNavigationItem(
                     icon = { Icon(painterResource(id = item.icon), contentDescription = item.title, tint = Color.Unspecified) },
-                    label = { Text(text = item.title) },
+                    label = { Text(text = item.title, fontSize = 11.sp) },
```

Comment:
Also there was a small UI problem, in the dashboard activity the bottom nav text for Critiques was overflowing, so I reduced text size